content/zh-cn/docs/tutorials/module-create/springboot-and-sofaboot.md

```
@@ -18,7 +18,99 @@ weight: 200
+- 已安装 arkctl 工具
+- Java 8 或更高版本
+
+### 4. 使用步骤
+
+#### 4.1 运行命令
+首先go build编译项目 
+这一步会生成 `arkctl.exe`（Windows）或 `arkctl`（Linux/Mac）可执行文件。
+
+在命令行中执行以下命令：
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)